workflow/parse.py

```diff
@@ -0,0 +1,64 @@
+import os 
+import pandas as pd 
+import csv
+ 
+tabb = dict()
+tabb['strand'] = []
+tabb['binding_position'] = []
+tabb['binding_posterior'] = []
+tabb['pwm_id'] = []
+tabb['binding_region'] = []
+tabb['binding_sequence'] = []
+tabb['binding_energy'] = []
+tabb['fast_record'] =[] 
+
+print(snakemake.input)
+print(snakemake.params)
+
+filenameparam = str(snakemake.params)
+
+dirrs = snakemake.input
+outdir = str(snakemake.output)
+print(outdir)
```

Comment:
Please remove all such `print` checks in the final version. We want the code clean.